.github/scripts/update_release_version.py

```
@@ -0,0 +1,116 @@
+        const_path: Path to the integration constants file.
+        tag_name: Release tag name to write as the integration version.
+
+    Raises:
+        ValueError: Raised when ``const_path`` lacks a VERSION assignment.
+
+    """
+    const_text = const_path.read_text()
+    version_line = f"VERSION = {json.dumps(tag_name)}"
+    updated_text, replacements = VERSION_PATTERN.subn(version_line, const_text, count=1)
+    if replacements != 1:
+        raise ValueError(f"{const_path} does not contain a VERSION assignment")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)